README.md

```diff
@@ -4,6 +4,14 @@ This repo is a monorepo, which contains the pieces that make up the Universal Co
 application which anyone can clone and self-host as a way to serve the connect widget via a url which can then be loaded
 into an iframe.
 
+## Authentication
+
+The express endpoints exposed in these repositories don't provide any authentication. You will need to fork the repo to add your own authentication.
+
```

Comment:
Should we change this wording so it doesn't sound like it's required? Technically adding auth isn't required.

Something like, `You will need to fork the repo to add your own authentication, if desired.`

Reply:
Done